packages/flutter/lib/src/cupertino/spell_check_suggestions_toolbar.dart

```diff
@@ -111,7 +117,6 @@ class CupertinoSpellCheckSuggestionsToolbar extends StatelessWidget {
       }
     });
     editableTextState.hideToolbar();
-    editableTextState.renderEditable.selectWordEdge(cause: SelectionChangedCause.toolbar);
```

Comment:
I'm wondering why this didn't work as expected?

Reply:
This call was operating on the previous frame's TextEditingValue, so it didn't consider the `userUpdateTextEditingValue` call above.  It was selecting the edge of the old word before that change.